test/test_aqua.jl

```diff
@@ -9,6 +9,7 @@ using FillArrays
                   piracy=(broken=true,)
                   )
     # manual piracy check to exclude identityoperator
-    pirates = [pirate for pirate in Aqua.Piracy.hunt(QuantumOpticsBase) if pirate.name != :identityoperator]
+    # FIXME: also adjoint. Move StateVector def to QuantumInterface.
+    pirates = [pirate for pirate in Aqua.Piracy.hunt(QuantumOpticsBase) if pirate.name != :identityoperator && pirate.name != :adjoint]
```

Comment:
Is this outdated? I believe `StateVector` is already in `QuantumInterface`. The only new `adjoint` defined here does not seem to be a pirate.

Reply:
`Base.adjoint(a::StateVector) = dagger(a)` is defined in QOB in states.jl. Guess this ought to be in QuantumInterface.

Reply:
I am weirded out how this did not pop up when I was moving definitions to QuantumInterface. Thanks for taking care of it.

Reply:
Do you want me to move that out in a separate PR, or should we just merge https://github.com/qojulia/QuantumInterface.jl/pull/14 right now and have the compat section bumped up in this pull request, together with deleting the `adjoint` from QOB?

Reply:
I'm in favor of releasing QI with the adjoint fix. We can also do a separate PR for QOB to remove the duplicate.

Reply:
Oops, accidentally removed the method directly in master. Web VS Code makes this too easy!

Reply:
Done, release should be out in 15ish minutes. And yes, I really enjoy web VScode and have the same bad habit :D. I will rerun the tests on master after QI is out